docs/zh/reference/information_schema/tables.md

```diff
@@ -0,0 +1,33 @@
+---
+displayed_sidebar: "Chinese"
+---
+
+# tables
+
+`tables` 提供有关表的信息。
+
+`tables` 提供以下字段：
+
+| **字段**        | **描述**                                                     |
+| --------------- | ------------------------------------------------------------ |
+| TABLE_CATALOG   | 表所属的 Catalog 名称。                                      |
+| TABLE_SCHEMA    | 表所属的数据库名称。                                         |
+| TABLE_NAME      | 表名。                                                       |
+| TABLE_TYPE      | 表的类型。 有效值：`BASE TABLE` 和 `VIEW`。                  |
+| ENGINE          | 表的引擎类型。 有效值：`StarRocks`、`MySQL`、`MEMORY` 和空字符串。 |
+| VERSION         | 该字段暂不可用。                                             |
+| ROW_FORMAT      | 该字段暂不可用。                                             |
+| TABLE_ROWS      | 表的行数。                                                   |
+| AVG_ROW_LENGTH  | 表的平均行长度（大小），等于 `DATA_LENGTH` / `TABLE_ROWS`。 单位：Byte。 |
+| DATA_LENGTH     | 数据长度（大小）。 单位：Byte。                              |
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/3cef0593-53f8-4f0d-8336-4266a4ff8f50)
remove the space